src/utils/helpers.js

```diff
@@ -133,6 +133,33 @@ const helpers = {
   getFullExplorerURL(urlPath) {
     return path.join(this.getExplorerURL(), urlPath);
   },
+
+  /**
+   * Compare 2 version strings with format X.X.X
+   * (only numbers and length must be 3)
+   * Works as a "greater than" operator
+   *
+   * @param {string} v1, first version string
+   * @param {string} v2, second version string
+   *
+   * @returns {Number} 0 if equal, 1 if v1 > v2 and -1 if v2 > v1
+   *
+   * @memberof Helpers
+   * @inner
+   */
+  cmpVersionString (v1, v2) {
```

Comment:
Can you remove the space between the method name and the parentesis?

Reply:
Done in [86d6ee0](https://github.com/HathorNetwork/hathor-wallet/pull/264/commits/86d6ee02563d136387db484d7848ea2da6a5a3a4)